src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -131,6 +132,8 @@ case class NIHStatus(
 
 object NIHStatus {
 
+  implicit val impNihStatus = jsonFormat7(NIHStatus.apply)
```

Comment:
Sweet I was hoping you'd ask this! So, my thought is that you should be able to use NIHStatus in just about any way you want (including JSON) simply by importing the object. You should not have to also import ModelJsonProtocol just because you decided you wanted to serialize the object.